### PR TITLE
add Genome::Sys::LockProxy::unlock_guard()

### PR DESCRIPTION
**Pull request by nnutter**
It doesn't really save a lot of code but it still seems like an improvement.

### CODE REVIEW

**Comment by acoffman** [comment]
It seems a little odd to have identical implementations in both Lock and LockProxy, but on the other hand, I'm not sure a single 2 line method is really worth adding an extra module and mixin for either. 

+1

File: lib/perl/Genome/Sys/LockProxy.pm [hunk 1]
```diff
@@ -8,6 +8,8 @@ use Genome::Sys::Lock qw();
 use Params::Validate qw(validate_with);
 use Scalar::Util qw(blessed);
 
+require Scope::Guard;
```
Comment (dufeiyu) [inline lib/perl/Genome/Sys/LockProxy.pm]: Does LockMigrationProxy.pm require Scope::Guard too ?

Comment (nnutter) [inline lib/perl/Genome/Sys/LockProxy.pm]: > Does LockMigrationProxy.pm require Scope::Guard too ?

Yep!  Added.

**Comment by dufeiyu** [comment]
+1  assume tests pass

**Pull request merged**